### ISSUE TITLE
Odd collision on Piglin Flag

### ISSUE THREAD

**Issue by FewerFlaws** [Globox1997/AdventureZ#13]
If I jump at the piglin flag I can stand on the thicker part of the wooden pole.  If I fall down on it from above I can phase right through the shaft.  It feels a bit clunky to be around >_<
![2021-01-07_13 52 15](https://user-images.githubusercontent.com/75643699/103949235-ce17bc80-50ef-11eb-8d88-0a70ec84eb7a.png)

**Comment by Globox1997** [comment]
Yes it definitely is kinda odd but that's how mc blocks are when they are bigger than one block

**Comment by FewerFlaws** [comment]
If it were me I would make them behave like either a fence or a block.  The ability to pass through the shaft is not a problem because many minecraft objects lack collision.  Right now the shaft impedes your movement if you leap onto the base from the ground, but lacks collision if you fall onto the base from above.  That's problematic.

Behaving like a fence would mean you can't jump onto the base but could still pass through the shaft if you fell onto it from above.  Behaving like a block would mean that you can jump onto the base from the ground.  Both of these behaviours are more consistent with vanilla =]

**Comment by Globox1997** [comment]
Changed the flag collission with [https://github.com/Globox1997/AdventureZ/commit/4deaeea44fad9ba847d1c81137c4e7825a94cb97](https://github.com/Globox1997/AdventureZ/commit/4deaeea44fad9ba847d1c81137c4e7825a94cb97) as you suggested.
